src/bin/cargo-fmt.rs

```diff
@@ -288,16 +288,16 @@ fn format_files(files: &[PathBuf],
         println!("");
     }
     let mut command = try!(Command::new("rustfmt")
-        .stdout(stdout)
-        .args(files)
-        .args(fmt_args)
-        .spawn()
-        .map_err(|e| match e.kind() {
-            std::io::ErrorKind::NotFound => {
-                std::io::Error::new(std::io::ErrorKind::Other,
-                                    "Could not run rustfmt, please make sure it is in your PATH.")
-            }
-            _ => e,
-        }));
+                               .stdout(stdout)
```

Comment:
I think this does not look good, though tolerable since the block indented closure is the last child of the chain. 
There is already a related issue (#1078).